src/main/java/org/fundacionjala/pivotal/core/api/APICommons.java

```diff
@@ -47,24 +57,36 @@ public static String getElement(final Response res, final String element) {
      * @param response the response from request.
      */
     public static void saveResponse(final String variableName, final Response response) {
-        requestResponse = new HashMap<>();
+        //requestResponse = new HashMap<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)